tests/test_clinvar_to_evidence_strings.py

```diff
@@ -54,8 +54,9 @@ def test_consequence_type_dict(self):
 class CreateTraitTest(unittest.TestCase):
```

Comment:
GetMappingsTest is for checking the mappings have been loaded from the files correctly, CreateTraitTest is for checking a trait object is created correctly. I can create a test for both of these, since it would affect both situations.